app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb

```diff
@@ -76,6 +80,11 @@ def tags_by_crn(crn)
     connection.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
   end
 
+  def resources
+    resource_list = connection.cloudtools.resource.controller.collection(:list_resource_instances)
+    resource_list.select { |resource| resource[:resource_plan_id].match(/databases-for-*/)}
```

Comment:
Deployed databases are identified by the resource controller api via resource plan ids that follow a format `/databases-for-*/`

Reply:
FYI if you just care about a boolean you can use `.match?(/regex/)`